File: LahorAppFrontend/src/app/Model/Obavijest.ts
import {DatePipe} from "@angular/common";


export class Obavijest{
  public id:Number;
  public naslov:string=" ";
  public autorId:Number=0;
  public sadrzaj:string=" ";
  public javnaObavijest:boolean=false;
  public slika:string=" ";
  public datumKreiranja:any;
  public prikazi:boolean=false;

  constructor(id:Number,naslov:string,autorId:Number,sadrzaj:string,javnaObavijest:boolean,
              slika:string,datumKreiranja:any,prikazi:boolean) {

    this.id=id;
    this.naslov=naslov;
    this.autorId=autorId;
    this.sadrzaj=sadrzaj;
    this.javnaObavijest=javnaObavijest;
    this.slika=slika;
    this.datumKreiranja=datumKreiranja;
    this.prikazi=prikazi;
  }
}

File: LahorWebApp/Data/Models/Obavijest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Data.Models
{
    [Table("Obavještenja")]
    public class Obavijest
    {
        [Key]
        public int Id { get; set; }
        public string Naslov { get; set; }
        public DateTime DatumKreiranja { get; set; }

        [ForeignKey(nameof(Autor))]
        public int AutorId { get; set; }
        public UpravnoOsoblje Autor { get; set; }
        public string SlikaObavještenja { get; set; }
        public string Sadrzaj { get; set; }
        public bool JavnaObavijest { get; set; }
    }
}
